boards/stm32f4discovery/Makefile.include

```diff
@@ -11,6 +11,7 @@ include $(RIOTMAKE)/tools/serial.inc.mk
 
 export DEBUG_ADAPTER ?= stlink
 export STLINK_VERSION ?= 2
+export OPENOCD_CONFIG := $(RIOTBOARD)/$(BOARD)/dist/openocd.cfg
```

Comment:
I do not like that you need here to set it to the default openocd value.
I would prefer it handled in the `stlink-adapter`, see below.